lib/src/collection_util.dart

```diff
@@ -1,34 +0,0 @@
-import 'booleans.dart';
-import 'exceptions.dart';
-
-/// Utilities for collection classes (`Iterable` and `Map`)
-///
-/// `since 0.4.0`
-class CollectionUtil {
-  CollectionUtil._();
-
-  /// Returns the value at [path] in a nested collection.
-  /// Takes a [path] `List` and a collection [coll].
-  /// Traverses `Map` (by key), `List` (by index), and `Iterable` (by index) values.
-  ///
-  /// `since 0.4.0`
-  static dynamic getIn(List<dynamic> path, dynamic coll) {
```

Comment:
IMO if this function became `static R getIn<R>(List<Object> path, Object coll) {` it would be no different to a JsonPath style function.... but you wanted it gone, so it's gone